testsuite/module-publish/src/packages/framework_usecases/sources/resource_groups_example.move

```diff
@@ -0,0 +1,201 @@
+                    value: 0,
+                    name,
+                };
+                move_to(owner, resource);
+            }
+        } else if (index == 2) {
+            if (exists<ExampleResource2>(owner_address)) {
+                let resource = borrow_global_mut<ExampleResource2>(owner_address);
+                resource.name = name;
+            } else {
+                let resource = ExampleResource2 {
+                    value: 0,
+                    name,
+                };
+                move_to(owner, resource);
+            }
+        } else if (index == 3) {
+            if (exists<ExampleResource3>(owner_address)) {
+                let resource = borrow_global_mut<ExampleResource3>(owner_address);
+                resource.name = name;
+            } else {
+                let resource = ExampleResource3 {
+                    value: 0,
+                    name,
+                };
+                move_to(owner, resource);
+            }
+        } else if (index == 4) {
+            if (exists<ExampleResource4>(owner_address)) {
+                let resource = borrow_global_mut<ExampleResource4>(owner_address);
+                resource.name = name;
+            } else {
+                let resource = ExampleResource4 {
+                    value: 0,
+                    name,
+                };
+                move_to(owner, resource);
+            }
+        } else if (index == 5) {
+            if (exists<ExampleResource5>(owner_address)) {
+                let resource = borrow_global_mut<ExampleResource5>(owner_address);
+                resource.name = name;
+            } else {
+                let resource = ExampleResource5 {
+                    value: 0,
+                    name,
+                };
+                move_to(owner, resource);
+            }
+        } else if (index == 6) {
+            if (exists<ExampleResource6>(owner_address)) {
+                let resource = borrow_global_mut<ExampleResource6>(owner_address);
+                resource.name = name;
+            } else {
+                let resource = ExampleResource6 {
+                    value: 0,
+                    name,
+                };
+                move_to(owner, resource);
+            }
+        } else if (index == 7) {
+            if (exists<ExampleResource7>(owner_address)) {
+                let resource = borrow_global_mut<ExampleResource7>(owner_address);
+                resource.name = name;
+            } else {
+                let resource = ExampleResource7 {
+                    value: 0,
+                    name,
+                };
+                move_to(owner, resource);
+            }
+        };
+    }
+
+    public entry fun set_p(_fee_payer: &signer, owner: &signer, index: u64, name: String) acquires ExampleResource0, ExampleResource1, ExampleResource2, ExampleResource3, ExampleResource4, ExampleResource5, ExampleResource6, ExampleResource7 {
```

Comment:
What's the use of `_fee_payer` as the field is not used?

Reply:
First argument is always the signer of the transaction, and transaction fee is charged from that account in the prologue.